_1327/information_pages/views.py

```diff
@@ -51,6 +54,11 @@ def edit(request, title, new_autosaved_pages=None):
 		messages.success(request, _("Successfully saved changes"))
 		return HttpResponseRedirect(reverse('information_pages:view_information', args=[document.url_title]))
 	else:
+
+		minutes = get_objects_for_user(request.user, MinutesDocument.VIEW_PERMISSION_NAME, klass=MinutesDocument)
+		information_documents = get_objects_for_user(request.user, InformationDocument.VIEW_PERMISSION_NAME, klass=InformationDocument)
+		polls = get_objects_for_user(request.user, Poll.VIEW_PERMISSION_NAME, klass=Poll)
+
```

Comment:
It seems like a bad idea to load these all the time. Can't verify the JavaScript until you fix the diff, however. I'm not gonna read through that entire file to figure out what you actually changed.